t/web/download.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+require 'test_helper'
+require_relative '../../app'
+
+describe 'Viewer' do
+  subject { Nokogiri::HTML(last_response.body) }
+
+  describe 'when viewing the home page' do
+    before { get '/united-states-of-america/download.html' }
+
+    it 'should have show some text' do
+      last_response.body.must_include 'https://cdn.rawgit.com/everypolitician/everypolitician-data/'
+      last_response.body.must_include '/data/United_States_of_America/'
+      last_response.body.must_include 'House/names.csv'
+      last_response.body.must_include 'Senate/names.csv'
+    end
```

Comment:
I suspect what you actually want to be testing here is something `subject.css('a.button--download/@href').map(&:value).select { |h| h.include? '/names.csv' }`